src/main/java/org/apache/commons/imaging/formats/tiff/datareaders/DataReaderTiled.java

```diff
@@ -85,26 +85,24 @@ private void interpretTile(final ImageBuilder imageBuilder, final byte[] bytes,
         if (sampleFormat == TiffTagConstants.SAMPLE_FORMAT_VALUE_IEEE_FLOATING_POINT) {
             // tileLength: number of rows in tile
             // tileWidth:  number of columns in tile
-            final int i0 = startY;
```

Comment:
Didn't search any specification document, but I would probably leave these variables as-is as they appear to be the type of vars copied from some existing code or document...

Reply:
Done